arangod/IResearch/IResearchView.cpp

```diff
@@ -314,7 +314,7 @@ arangodb::Result persistProperties(
   if (!engine->inRecovery()) {
     // change view throws exception on error
     try {
-      engine->changeView(view.vocbase(), view.id(), view, true);
+      engine->changeView(view.vocbase(), view, true);
```

Comment:
should changeView take a TRI_vocbase_t separately?

Reply:
I dunno the ID's would have been also enough, but I don't think it matters. Most places get the vocbase passed in

Reply:
if you pass in a DataSource then it already has a vocbase, so it seems redundant
I understand that was the original API, but might as well clean it up